scripts/models/TownHall.js

```diff
@@ -45,11 +45,7 @@
       var title;
       if (this.district) {
         //state leg
-        var districtType = this.district.split('-')[0];
-        var districtNo = this.district.split('-')[1];
-
-        title = constants[districtType];
-        this.displayDistrict = title + ' ' + districtNo;
+        this.displayDistrict = this.District;
       } else {
```

Comment:
we are eventually going to get rid of storing `District` in the database, and do the formatting on the frontend. So this isn't a helpful fix.

Reply:
Gotcha.  I'll fix it in a few hrs.